internal/cmd/gocdk/biome_add.go

```diff
@@ -16,30 +16,39 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
 	"path/filepath"
 
+	"github.com/spf13/cobra"
 	"golang.org/x/xerrors"
 )
 
-func biomeAdd(ctx context.Context, pctx *processContext, args []string) error {
-	// TODO(clausti) interpolate launcher from one supplied as a flag
-	f := newFlagSet(pctx, "biome")
-	const usageMsg = "gocdk biome add"
-	if err := f.Parse(args); xerrors.Is(err, flag.ErrHelp) {
-		return nil
-	} else if err != nil {
-		return usagef("%s: %w", usageMsg, err)
-	}
-	if f.Arg(0) != "add" {
-		return usagef("%s: expected add, got %v", usageMsg, f.Args()[0])
+// TODO(rvangent): This should probably not be in biome_add.go anymore.
```

Comment:
Are you thinking we should revisit having a "biome" dir for the biome commands (`add` and `list` at least, maybe eventually `clone` or others) to live in? Or just rename the file?

Reply:
We should discuss; it should be consistent for all commands, not just `biome`. But it's pretty clear that `biome_add.go` is not the right filename given that it will have other `biome` subcommands.

Options include subdirectories/packages, `biome.go` with all of the `biome` subcommands, `biome.go` that sets up the commands and separate `biome_foo.go` for each subcommands, ...?